t/venom/vars/all.yml

```diff
@@ -72,3 +74,6 @@ dot1x_eap_peap.profiles.wired.id: catch_dot1x_wired_eap_peap_mschapv2
 dot1x_eap_peap.profiles.wired.description: 802.1X wired EAP-PEAP-MSCHAPv2
 dot1x_eap_peap.profiles.wired.filters.connection_type: Ethernet-EAP
 dot1x_eap_peap.profiles.wired.filters.connection_sub_type: MS-EAP-Authentication
+mac_authentication.profiles.wired.id: catch_wired_mac_authentication
+mac_authentication.profiles.wired.description: Wired Mac authentication with unreg_on_accounting_stop
+mac_authentication.profiles.wired.filters.connection_type: Ethernet-NoEAP
```

Comment:
All variables related to this scenario should be prefixed by `wired_mac_auth` in place of `mac_authentication`